src/templates/landing_pages/introducing_usb.jsx

```diff
@@ -0,0 +1,356 @@
+import React from 'react';
+import Layout from './_common/layout.jsx';
+
+const UsbPage = () => (
+    <Layout
+        meta_description={it.L('[_1] Introducing USB', it.broker_name)}
+        css_files={[
+            'https://style.binary.com/binary.css',
+            'https://style.binary.com/binary.more.css',
+            it.url_for('css/usbstyle.css'),
+        ]}
+        js_files={[
+            'https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js',
+            'https://cdnjs.cloudflare.com/ajax/libs/jquery-easing/1.3/jquery.easing.min.js',
+            'https://style.binary.com/binary.more.js',
+            'https://academy.binary.com/js/animate.js',
+            it.url_for('js/landing_pages/common.js'),
+            it.url_for('js/landing_pages/usb_page.js'),
+        ]}
+    >
+
+        <div id='page-top' />
+        <nav className='navbar navbar-custom navbar-fixed-top' role='navigation'>
+            <div className='container-fluid'>
+                <div className='row'>
+                    <div className='col-xl-3'>
+                        <div className='logo-wrapper'>
+                            <a className='logo-parent logo-scroll' href='#page-top'>
+                                <div className='logo'>
+                                    <div />
+                                </div>
+                                <div className='mobile-hide binary-logo-text'>
+                                    <div />
+                                </div>
+                            </a>
+                        </div>
+                    </div>
+                    <div className='col-xl-9'>
+                        <div id='push' className='grill grill-ico' />
+                        <div className='row'>
+                            <div className='col'>
+                                <div id='push-nav' className='topnav'>
+                                    <a className='page-scroll' href='#video'>{('Overview')}</a>
```

Comment:
if you don't want translations, you don't need this brackets at all. you can just do something like `<a className='page-scroll' href='#video'>Overview</a>`

Reply:
Actually, we do since ESlint has max length 120